plenum/server/node.py

```diff
@@ -1856,18 +1995,29 @@ def startViewChange(self, proposedViewNo: int):
         :param proposedViewNo: the new view number after view change.
         """
```

Comment:
This cannot be implemented quickly, so I don't think we should do it within this task - I'm adding TODO